src/components/menu/js/menuServiceProvider.js

```diff
@@ -476,7 +480,7 @@ function MenuProvider($$interimElementProvider) {
         top: Math.round(position.top),
         left: Math.round(position.left),
         // Animate a scale out if we aren't just repositioning
-        transform: !opts.alreadyOpen ? $mdUtil.supplant('scale({0},{1})', [scaleX, scaleY]) : undefined,
+        transform: !opts.alreadyOpen && !opts.isAnimating ? $mdUtil.supplant('scale({0},{1})', [scaleX, scaleY]) : undefined,
```

Comment:
I worry that this is not a fix but a bandaid. The issue is not the animation but the scale X/Y calculations when scrolled right ?